OpenSim/Simulation/Model/Blankevoort1991Ligament.cpp

```diff
@@ -62,6 +44,11 @@ Blankevoort1991Ligament::Blankevoort1991Ligament(
     set_slack_length(slack_length);
 }
 
+GeometryPath& Blankevoort1991Ligament::initGeometryPath() {
```

Comment:
`init` methods on classes are *usually* a code smell - unless you're writing like a game developer with exceptions turned off or something.

Reply:
I'm trying to find a way to support creating new objects that own paths (of any type) that is scripting friendly. I'm not sure that this is the best approach. Another approach could be the pointer adoptee approach used elsewhere in OpenSim (although I'm not sure how to update the `path` property in that case).